run.py

```diff
@@ -6,38 +6,46 @@
 '''
 
 import os
-import datetime
 import json
-import flask
-import linebot
-import linebot.models.events
-import linebot.exceptions
-import linebot.models
-import linebot.models.events
-import Monster.Utils
-import Monster.Drama
+from Monster import Utils
+from Monster import Drama
+from datetime import datetime
+from flask import Flask
+from flask import request
+from flask import abort
+from linebot import LineBotApi
+from linebot import WebhookHandler
+from linebot.exceptions import LineBotApiError
+from linebot.exceptions import InvalidSignatureError
+from linebot.models import TextMessage
+from linebot.models import ImageMessage
+from linebot.models import AudioMessage
+from linebot.models import TextSendMessage
+from linebot.models.events import FollowEvent
+from linebot.models.events import MessageEvent
+from linebot.v3.messaging import MessagingApiBlob
 
 config_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '.', 'config')
 config_path = os.path.join(config_dir, 'linebot.conf')
 line_bot_config = json.load(open(config_path, 'r', encoding='utf8'))
 
-current_date = datetime.datetime.today().strftime('%Y%m%d')
+current_date = datetime.today().strftime('%Y%m%d')
 
-LINE_BOT_API = linebot.LineBotApi(line_bot_config['CHANNEL_ACCESS_TOKEN'])
-HANDLER = linebot.WebhookHandler(line_bot_config['CHANNEL_SECRET'])
-USER_LOG_PATH = './log/' + current_date
+LINE_BOT_API = LineBotApi(line_bot_config['CHANNEL_ACCESS_TOKEN'])
+HANDLER = WebhookHandler(line_bot_config['CHANNEL_SECRET'])
+USER_LOG_PATH = os.path.join(".", "log", current_date)
```

Comment:
This is actually the better method to reach out to the config or the path may lose.